src/app/account-settings-page/account-settings-page.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import {UserService} from "../../services/user.service";
+import {AccountService} from "../../services/account.service";
+import {User} from "../../model/user";
+import {ModalService} from "../../services/modal.service";
+import {ModalAlertContentComponent} from "../modal-alert-content/modal-alert-content.component";
+import {Router} from "@angular/router";
```

Comment:
It's not a big deal, but these import statements should use single quotes so that they are consistent with the rest of the app. I believe you can modify your IDE's settings so that automatic imports use a single quote instead.